src/iop/dither.c

```diff
@@ -160,8 +157,12 @@ static inline float _rgb_to_gray(const float *const restrict val)
 #ifdef _OPENMP
 #pragma omp declare simd
 #endif
-static inline void nearest_color(float *const restrict val, float *const restrict err, int graymode,
-                                 const float f, const float rf)
+static inline void nearest_color(
```

Comment:
This function should also get a leading underscore since you're changing things to be consistent with dt's style.

Reply:
:-)